modules/reviewing-pod-status.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * support/investigating-pod-issues.adoc
+
+[id="reviewing-pod-status_{context}"]
+= Reviewing Pod status
+
+Query Pod status and error states. Query a Pod's associated deployment configuration and review base image availability.
```

Comment:
I'm not sure of others have differing opinions, but this seems a little incomplete/abrupt to me.

Suggest "You can query a Pod's status and error states"

Other instances in the PR, if you choose to update.

Reply:
Thanks Andrea. Reading this back I completely agree. I have now reviewed this throughout the PR and made changes where appropriate.